apps/sms/views/conversation/style/composer.css

```diff
@@ -4,12 +4,6 @@
   background: #e1f0ec;
```

Comment:
For the `#composer-container`/ `.flex-composer` / `#messages-contact-pick-button`, I still keep theses in shared composer.css because they are sharing the same styling between 2 views, but maybe we can put these in more general place like `sms.css`?